src/web/modules/payment_links/payment_links.http.ts

```diff
@@ -58,6 +58,15 @@ export async function listCSV(req: Request, res: Response, next: NextFunction):
   }
 }
 
+export function search(req: Request, res: Response) {
+  res.render('payment_links/search', { csrf: req.csrfToken() })
+}
+
+export function searchRequest(req: Request, res: Response) {
+  const id = req.body.id.trim()
+  res.redirect(`/payment_links/${id}`)
```

Comment:
Does this work at the moment? As in, does it retrieve the payment link from products?

Reply:
Yep, that's an existing page to get a payment link from products and render a page displaying all the details. Currently you can get to it from the page that lists all payment payment links or payment links for a gateway account.